officemanager/department/migrations/0002_auto_20190904_1052.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.4 on 2019-09-04 10:39
+
+from django.db import migrations
+
+def add_titles_dep(apps, schema_editor):
+    title_list = [
+        'Business Intelligence',
+        'IT Management',
+        'Administration',
+        'IT Procurement',
+        'IT Security',
+        'Network Adminstration',
+        'Systems Analyst & Architecture',
+        'User Support & Services'
+    ]
+
+    for line in title_list:
+        Dep = apps.get_model('department', 'Departments')
```

Comment:
I think it makes sense to get the model before the loop once, and not at each step of the loop.